docs/1-about-ergodex/intro.md

```diff
@@ -0,0 +1,22 @@
+---
+sidebar_position: 0
+---
+
+# Introduction
+
+## What is ErgoDEX?
+ErgoDEX is a non-custodial decentralized exchange that allows a quick, effortless, and secure 
+transfer of liquidity inside and between the Ergo and Cardano networks. The eUTXO model gives the unique possibility to have shared 
+liquidity among different networks. ErgoDEX will eventually feature a full AMM, Order Book and more.
+
+ErgoDEX is an open-source DeFi product which take a part in both Ergo and Cardano platforms and as a 
+consequence exists for purposes of both communities. You can imagine ErgoDEX infrastructure like the following:
+
+- **ErgoLabs:** The team of developers who is developing and maintaining ErgoDEX protocol, user interface and other 
+  products;
+- **The ErgoDEX Protocol:** a set of smart contracts that create an automated market maker and order book;
+- **ErgoDEX User Interface (UI):** Web (or mobile) interface which provides access in conjunction with the protocol;
+- **ErgoDEX Execution Bots:** off-chain software which function is to execute DEX orders and which works in completely 
```

Comment:
Off-chain software whose function is to execute DEX orders and  works in a completely